src/mongocxx/options/find_one_and_replace.hpp

```diff
@@ -139,33 +156,13 @@ class MONGOCXX_API find_one_and_replace {
     ///
     const stdx::optional<bool>& upsert() const;
 
-    ///
-    /// Sets the bypass_document_validation option.
-    /// If true, allows the write to opt-out of document level validation.
-    ///
-    /// @note
-    ///   On servers >= 3.2, the server applies validation by default. On servers < 3.2, this option
-    ///   is ignored.
-    ///
-    /// @param bypass_document_validation
-    ///   Whether or not to bypass document validation
-    ///
-    void bypass_document_validation(bool bypass_document_validation);
-
-    ///
-    /// Gets the current value of the bypass_document_validation option.
-    ///
-    /// @return The optional value of the bypass_document_validation option.
-    ///
-    const stdx::optional<bool>& bypass_document_validation() const;
```

Comment:
These moved because I had forgotten that I had added them in the agg/bulk opps bypass doc validation pr, then I added them again here in the proper alphabetical order, then I rebased...

I can put them back if you'd prefer? I like them better arranged by name though, feels more consistent.